.github/workflows/check-release-notes.yml

```
@@ -0,0 +1,18 @@
+# Checks Release Notes in the PR description when:
+#   opened - PR is opened
+#   edited - The title/description are changed
+#   synchronize - New commits appeared.
+#                 We don't use the new content, but we still need to mark this commit Green/Red in GitHub UI
+
+name: Check Release Notes in the description
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  check:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: JetBrains/compose-multiplatform/tools/changelog/check-release-notes-github-action@igordmn-patch-7
+        with:
+          checkout_ref: 'igordmn-patch-7'
```

```suggestion
      - uses: JetBrains/compose-multiplatform/tools/changelog/check-release-notes-github-action@master
```

- merge https://github.com/JetBrains/compose-multiplatform/pull/5251 first
- apply this fix
- merge this PR